In-App Links in Channel's Bookmarks (#8236)
mattermost/docs: source/end-user-guide/collaborate/manage-channel-bookmarks.rst

Opening a channel bookmark works the same way as selecting a file link or attachment in a message. Select or tap a bookmark to view the file or link.

> .. note::

Author: ```suggestion
.. tip::
```

>    **Desktop app users** (Mattermost v10.11+): When you click bookmarks with ``mattermost://`` URLs in the desktop app, they open directly within the app using internal deep linking. This provides a seamless experience without external protocol prompts that were shown in earlier versions.

Author: ```suggestion
   From Mattermost v10.11, when you select bookmarks with ``mattermost://`` URLs in the desktop app, those links open directly within the desktop app using internal deep linking.
```
